library/src/scripts/features/search/IndependentSearch.tsx

```diff
@@ -80,6 +81,8 @@ export function IndependentSearch(props: IProps) {
                 hideSearchButton={props.hideSearchButton}
                 contentClass={props.contentClass}
                 valueContainerClasses={props.valueContainerClasses}
+                disableAutocomplete={props.disablePointerEvent ? true : false}
```

Comment:
You can do this in the prop, no need to put the logic in the parent. It makes it more "portable" that way.

Reply:
Fix- move logic to searchBar component